app/controllers/visualizations_controller.rb

```diff
@@ -339,17 +352,14 @@ def displayVis
 
     rel_vis = which_vis(has_time_data, has_loc_data, has_pics, field_count, format_data)
 
-    # A list of all current visualizations
-    all_vis =  ['Map', 'Timeline', 'Scatter', 'Bar', 'Histogram', 'Pie', 'Table', 'Summary', 'Photos']
-
     # Defaut vis if one exists for the project
     default_vis = @project.default_vis.nil? ? 'none' : @project.default_vis
 
     # The finalized data object
     @data = { projectName: @project.title,   projectID: @project.id,
               fields: data_fields,           dataPoints: format_data,
               metadata: metadata,            relVis: rel_vis,
-              allVis: all_vis,               defaultVis: default_vis,
+              allVis: @all_vis,               defaultVis: default_vis,
```

Comment:
Need to be able to change the vis with a query string in displayVis as well as show